cucumber/authenticators/features/support/authn_oidc_helper.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+#
+# Utility methods for OIDC authenticator
+#
+module AuthnOidcHelper
+  include AuthenticatorHelpers
+
+  # relevant for original oidc flow
```

Comment:
We should just remove these dead comments.

Reply:
Those comments are relevant to our previous OIDC use case (you can look also on the disabled feature authn_oidc_login.feature_disable) which we decided to save meanwhile in the code to be able to continue this feature in the near future